.gitignore

```diff
@@ -0,0 +1,4 @@
+.gradle/
+build/
+.idea/
+out/
```

Comment:
Nitpick but do you mind adding a newline to keep github happy?

Reply:
Sure thing! Missed it.